tests/test_config_parser.py

```diff
@@ -378,6 +378,8 @@ def _update_ref_from_file(self, mk_config: Dict[str, Any], job_name: str = DEFAU
                 self._append_list("extra_search_paths", mk_config_host["ExtraSearchPaths"], job_name, ":")
             if "QemuHeadless" in mk_config_host:
                 self.reference_config[job_name]["qemu_headless"] = mk_config_host["QemuHeadless"]
+            if "QemuArgs" in mk_config_host:
+                self._append_list("qemu_args", mk_config_host["QemuArgs"], job_name, " ", True)
```

Comment:
Please don't just pass `True`, use named arguments so we know what we're setting to `True`